Makefile

```diff
@@ -0,0 +1,2 @@
+all:
```

Comment:
Opinion is divided as to whether Go projects should use Makefiles. I'm personally a huge fan of `make` for basically everything except Go. One reason is that Go already has its own automated build tooling, so you can just run `go get ...` some project, it will fetch, build, and install it for you.

As soon as you require a Makefile for your project to build, then `go get` no longer works, which will puzzle some people and irritate others. There are some complicated builds which do need scripting, such as running `go generate` or compiling C dependencies, and a Makefile is a regrettable necessity for those. If we _don't_ need a Makefile, we shouldn't supply one.

Reply:
Very interesting. I've been unable to compile it using `go get github.com/tomdoherty/cidrgrep` doesn't seem to work for me just yet. maybe once merged to master?